src/config-import.js

```diff
@@ -215,6 +285,53 @@ async function findPresetsFile(entries) {
   return { presets, fields }
 }
 
+/**
+ * @param {ReadonlyArray<Entry>} entries
+ */
+async function findTranslationsFile(entries) {
+  const translationEntry = entries.find(
+    (entry) => entry.filename === 'translations.json'
+  )
+
+  if (!translationEntry) return
+
+  /** @type {unknown} */
+  let result
+  try {
+    result = await json(await translationEntry.openReadStream())
+  } catch (err) {
+    throw new Error('Could not parse translation.json')
+  }
+  assert(isRecord(result), 'Invalid translations.json file')
+  return parseTranslations(result)
+}
+
+/**
+ * @param {TranslationsFile} translations
+ * @returns {Translations}
+ */
+function parseTranslations(translations) {
+  /** @type {Translations} */
+  const translatedLangs = {}
+  for (let [lang, translationByLang] of Object.entries(translations)) {
+    const langTranslation = { presets: {}, fields: {} }
+    if (!isRecord(translationByLang))
+      throw new Error('invalid translation lang object')
+    for (let [docType, translationByDocType] of Object.entries(
+      translationByLang
+    )) {
+      if (!isRecord(translationByDocType))
+        throw new Error('invalid translation docType object')
+      if (!(docType === 'presets' || docType === 'fields')) continue
```

Comment:
Should we error or warn in this case?

Reply:
I think we should warn but maybe after we update the default config so that it doesn't have `categories` (I can leave a TODO comment that addressed this, but for know we just `continue`?)